models/surveys.go

```diff
@@ -15,9 +17,9 @@ import (
 	"strings"
 	"unicode"
 
+	"github.com/gofrs/uuid"
```

Comment:
Any reason for choosing this `uuid` package?

Reply:
https://github.com/satori/go.uuid/issues/84

It's a fork of this repo that's maintained and has the same interface.

Reply:
That's fair